css/properties/font-variant.json

```diff
@@ -158,43 +158,44 @@
             "description": "Greek accented characters",
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": true
               },
               "chrome_android": {
-                "version_added": null
+                "version_added": true
               },
               "edge": {
-                "version_added": null
+                "version_added": "12",
+                "version_removed": "18"
```

Comment:
This looks weird to me. Can you explain why you've added "version_removed": "18" here?

Reply:
I was about to give a proper reason why, but after looking at it again, I just realized that I might be wrong and looking at the behavior a little too closely.  Basically, I tested with `small-caps` in Edge 17 and 18, and this is the result I got from both (top part being the original, bottom being the one with `font-variant: small-caps`):

17: <img width="109" alt="Edge 17" src="https://user-images.githubusercontent.com/5179191/56724788-85905380-6700-11e9-98b7-676668baffac.png">

18: <img width="103" alt="Edge 18" src="https://user-images.githubusercontent.com/5179191/56724797-888b4400-6700-11e9-8a29-bf611bccf322.png">

Here's the test, if it helps!  https://codepen.io/anon/pen/BEGGLB?&editable=true

Reply:
I don't much about the topic, but it looks like the rendering changed a bit but the small-caps functionality still works? In that case, I would mark this as removed in 18

Reply:
Sure thing, removing it now!